SourceKitStressTester/Sources/Common/CompilerArgs.swift

```diff
@@ -53,7 +53,7 @@ public struct CompilerArgs {
           let fileRemoved = listArgs.filter { $0 == file.path }
           if fileRemoved.count != listArgs.count {
             let newFileList = tempDir.appendingPathComponent(UUID().uuidString)
-              .appendingPathExtension(".SwiftFileList")
+              .appendingPathExtension("SwiftFileList")
```

Comment:
Didn't mean to put this in here but I guess it doesn't really matter. `appendingPathExtension` adds the `.` so it had two before